tests/advanced_visibility_test.go

```diff
@@ -103,15 +102,9 @@ func (s *AdvancedVisibilitySuite) SetupSuite() {
 		// Allow the scavenger to remove any build ID regardless of when it was last default for a set.
 		dynamicconfig.RemovableBuildIdDurationSinceDefault.Key(): time.Microsecond,
 	}
+	s.FunctionalTestSuite.SetupSuiteWithDefaultCluster(testcore.WithDynamicConfigOverrides(dynamicConfigOverrides))
```

Comment:
can't you write this as
```suggestion
	s.SetupSuiteWithDefaultCluster(testcore.WithDynamicConfigOverrides(dynamicConfigOverrides))
```
?

Reply:
It meant to be `FunctionalTestSuiteBase`. I am explicitly indicate struct which this setup method belongs too.